### DISCUSSION TITLE
no detection even if red and blue boxes are presents

### DISCUSSION THREAD

**Discussion by manjump** [blakeblackshear/frigate#14764] [Ask A Question]
Hello,

Why there are not detections even if I see the blue and red boxes around me? I specify I test on the night. No problem on the day.
[code.txt](https://github.com/user-attachments/files/17612116/code.txt)

**Comment by NickM-27** [comment]
blue box usually means there is a detection. Can you show an example?

**Comment by manjump** [comment]
Yes sur.
![IMG_3864](https://github.com/user-attachments/assets/39054605-f0c0-4dd2-ab4f-34618d727be5)
![IMG_3865](https://github.com/user-attachments/assets/e3bb8af9-8187-4e03-a4f5-295e72c17552)
![IMG_3866](https://github.com/user-attachments/assets/154baaf1-321a-402f-8d92-22ee03bd2bec)
![IMG_3867](https://github.com/user-attachments/assets/2a057230-d3b4-4225-8b94-fe5f6c3d28c9)
![IMG_3868](https://github.com/user-attachments/assets/ecbcb2fa-909e-454c-a84e-5d70283b34ca)
![IMG_3869](https://github.com/user-attachments/assets/24818980-7ad7-45a0-b631-5e9c42e04330)

**Reply by NickM-27** [reply]
the person is not scoring high enough to be a true positive, you may need to reduce the threshold

**Comment by manjump** [comment]
Treshold = 10
Contour aera = 10
And nothing, no detect. It's strange because we see the blue box no

**Reply by NickM-27** [reply]
you are referring to motion settings, I am referring to object settings

**Comment by manjump** [comment]
Sorry but I don't understand what it means. It's a problem with my config.yaml?

**Reply by NickM-27** [reply]
see https://docs.frigate.video/configuration/object_filters

**Comment by tpjanssen** [comment]
I had the same in the debug view on a camera without any zones. Maybe this is the case on your side as well?

**Reply by manjump** [reply]
I'll try to understand the filters and I ll be back

**Reply by tpjanssen** [reply]
Just to be more precise, my detections were fine, only the object list remained empty.

**Reply by manjump** [reply]
your detections were fine, but have you the parameter in your config?
`filters:
        person:
          threshold: 0.XX`
Because me, it works the day, without this parameter in config, but doesn't work the night

**Comment by manjump** [comment]
If I have not filters in my config, normally frigate must detect every time no?

**Reply by NickM-27** [reply]
There are default values for the filters like it says in the docs

**Reply by manjump** [reply]
I read this page: https://docs.frigate.video/configuration/object_filters
But there is not default value
I search "threshold" with function search but nothing

**Reply by ElectronicBattle** [reply]
This is the part I used to set my own config:
![image](https://github.com/user-attachments/assets/b01fecb0-373f-40ed-a456-c4fcee78bd1c)

This config is the result:
![image](https://github.com/user-attachments/assets/28de3244-ac36-4bc8-97d9-3e976c24e8b0)

Isn't that what you need to play with?

**Comment by manjump** [comment]
Thanks Electronicbattle but the current score = (XX%)/100 ?? because I don't found this information in debug.

**Reply by ElectronicBattle** [reply]
Perhaps I do not understand your problem correctly, but I think your config file is incorrect.

You config is shown at https://github.com/user-attachments/files/17612116/code.txt

At the moment yours says
```
 objects:
      track:
        - person
        - dog
      filters:
        person:
          min_score: 0.50
```

but you have not set a threshold for person so you need it to say something like

```
 objects:
      track:
        - person
        - dog
      filters:
        person:
          min_score: 0.50
          threshold: 0.65
```
My config is written differently. I have a section at the beginning which defines the ```min_score``` and ```threshold``` globally for all cameras rather than doing it individually per camera which is how you prefer to do it.

I might not have followed your problem correctly, but I do think your config needs adjusting. I started with values of (say)

```
        person:
          min_score: 0.50
          threshold: 0.65
```

and ran Frigate for a few days and then adjusted it to (say)

```
        person:
          min_score: 0.75
          threshold: 0.85
```

and then repeated until I had the setting adjusted for minimum false alarms and best reliability.   It was an experimental process: try these values, then trim them, then try again, then trim again ...after a week or so it was perfect.

**Comment by manjump** [comment]
Hi, 
What is the minimum distance you can detect at night please?
And finally, the résolution (2K or hight) is it important for detect?

**Reply by NickM-27** [reply]
Depends on the camera, external light available, and the model

**Reply by manjump** [reply]
maybe a model please?
